server/app.js

```diff
@@ -1,13 +1,12 @@
 import express from "express";
-
 import apiRouter from "./api";
 import config from "./utils/config";
 import {
-	clientRouter,
-	configuredHelmet,
-	configuredMorgan,
-	httpsOnly,
-	logErrors,
+  clientRouter,
```

Comment:
It seems like you didn't change anything on this file, remove this change from the commit

Reply:
how do I do that

Reply:
I think this should revert the file to the same as master:

```
git checkout master server/app.js
```

Then you can just commit the file again

Reply:
done